requirements-dev.txt

```diff
@@ -0,0 +1 @@
+pytest==7.1.2
```

Comment:
Pinning the pytest version SGTM. What's the plan to keep this up-to-date? Are we going to set up dependabot to bump this for us?

Reply:
Yes.